### DISCUSSION TITLE
Browserslist: caniuse-lite is outdated - running "npx browserslist@latest --update-db" does not fix it

### DISCUSSION THREAD

**Discussion by FBosler** [vercel/next.js#21505] [Help]
Not sure if this is related to next.js, but I am getting the following warning:

```
Browserslist: caniuse-lite is outdated. Pleas run:
npx browserslist@latest --update-db
```

However, doing so does not fix the warning. Anybody else encountered something similar?

**Comment by tarunama** [comment]
I have encountered by CI test in PR.
https://github.com/vercel/next.js/pull/21484

```
test/integration/production-build-dir/test/index.test.js

Production Custom Build Directory > With basic usage > should render the page
```

Another PR was encountered the same error.
